handlers/dialog360/dialog360.go

```diff
@@ -621,7 +632,9 @@ func (h *handler) Send(ctx context.Context, msg courier.Msg, clog *courier.Chann
 
 		} else if i < len(msg.Attachments()) && (len(qrs) == 0 || len(qrs) > 3) {
 			attType, attURL := handlers.SplitAttachment(msg.Attachments()[i])
-			attType = strings.Split(attType, "/")[0]
+			splitedAttType := strings.Split(attType, "/")
```

Comment:
@norkans7 how about we rework this to call resolve media and then we can also check that the image is 512 x 512 ?

Reply:
Okay, I'll update that